app/assets/javascripts/admin/addon/controllers/admin-customize-themes-show.js

```diff
@@ -385,7 +386,16 @@ export default class AdminCustomizeThemesShowController extends Controller {
           this.allThemes.removeObject(model);
           this.transitionToRoute("adminCustomizeThemes");
         });
-      },
+      }
+    });
+  }
+
+
+  @action
+  showThemeSettingsEditor() {
+    this.dialog.alert({
```

Comment:
👍 on using the dialog service here. But using the `alert` option, gives us an extra "OK" button here: 

<img width="675" alt="image" src="https://github.com/discourse/discourse/assets/368961/5cce5781-322b-43b4-9183-64179ae81f02">

The dialog API also allows setting the buttons, and we can maybe use an empty array to remove them entirely?